types/platform.ts

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
looks like this a copy of `src/types/platform.ts`

Reply:
Fixed.